### DISCUSSION TITLE
Display Ads Reports "tactics" are required?

### DISCUSSION THREAD

**Discussion by ncoughlin** [amzn/ads-advanced-tools-docs#58] [Q&A]
I'm trying to download all the available data for Display Ads using the reporting API, and I have a question about the "tactic" filter.

It's my understanding based on this: [sponsored display docs](https://advertising.amazon.com/API/docs/en-us/sponsored-display/recommendations)

That there are really two options for this "T00020" and "T00030". So if I wanted to download all the data for display ads grouped by campaign I would have to make two separate report requests (one for each tactic). Is that correct? Or is there a way to request `tactic: all` the way we can with `creativeType: all`.

On a side note, it would be very useful if we could simply use `metrics: all` for metrics. Tracking the various metrics available for each report type is very tedious.

**Comment by elizafarley** [comment]
Hi @ncoughlin, that's correct you would need to request two separate reports, one for each tactic. This [FAQ](https://advertising.amazon.com/API/docs/en-us/reporting/v2/faq#why-arent-my-sponsored-display-reports-returning-data) might help. 

Thanks for the feedback on metrics. I'll pass that on to the product team.

**Reply by papaDhk** [reply]
Hello @elizafarley,
Does this apply to the reporting v3 API?

**Reply by elizafarley** [reply]
No, version 3 reporting returns all tactics in one report. There's more info on this in the [migration guide](https://advertising.amazon.com/API/docs/en-us/reference/migration-guides/reporting-v2-v3#sponsored-display-tactic).

**Reply by papaDhk** [reply]
Great! Thank you for the quick response!
